todo.js

```diff
@@ -0,0 +1,130 @@
+
+//DEFINITIONS
+var bullet = "- ";
+
+var button = $("<button>").attr("type", "submit").text("submit");
+var input = $("<input>").attr("type", "string");
+var label = $("<label>").append("Add an item <br>").append(this.input);
+
+var icon = function(src) {
+  this.alt = "delete item";
+  this.src = src;
+  this.width = "30px";
+  this.initialOpacity = 0.1;
+}
+
+//LOAD DATA// make items array corresponding to what is stored in localStorage
+var savedData = JSON.parse(localStorage.getItem("items"));
+
+if (!savedData) {
+  var items = [];
+} else {
+  var items = savedData;
+  for (var i = 0; i < savedData.length; i++) {
+      var $p = $("<p>")
+      var deleteIcon = new icon("https://cdn3.iconfinder.com/data/icons/google-material-design-icons/48/ic_delete_48px-128.png");
+      var $deleteIcon = $("<img>")
+        .attr("src", deleteIcon.src)
+        .attr("alt", deleteIcon.alt)
+        .attr("width", deleteIcon.width)
+        .attr("style", String("opacity: " + deleteIcon.initialOpacity))
+        .on("mouseenter", function(event) {
```

Comment:
you dont need to do this for styles. instead if the img element had a class like `<img class="delete-icon">` and in css you had 
```
.delete-icon {
   opacity: 0.1;
}

.delete-icon:hover {
   opacity: 1;
}
```
you get the same thing and it takes styles outside of js